refresh/templates/common/Package.swift

```diff
@@ -1,27 +1,36 @@
+// swift-tools-version:4.0
 import PackageDescription
 
 let package = Package(
     name: "<%- executableModule %>",
-    targets: [
-      Target(name: "<%- executableModule %>", dependencies: [ .Target(name: "<%- applicationModule %>") ]),
-<% if (sdkTargets.length > 0 || appType === 'crud') { -%>
-      Target(name: "<%- applicationModule %>", dependencies: [
-<% if (appType === 'crud') { -%>
-            .Target(name: "<%- generatedModule %>"),
-<% } _%>
-<% Object.keys(sdkTargets).forEach(function(target) { -%>
-            .Target(name: "<%- sdkTargets[target] %>"),
-<% }); -%>
-      ]),
-<% } _%>
-    ],
     dependencies: [
-        .Package(url: "https://github.com/IBM-Swift/Kitura.git",             majorVersion: 1, minor: 7),
-        .Package(url: "https://github.com/IBM-Swift/HeliumLogger.git",       majorVersion: 1, minor: 7),
-        .Package(url: "https://github.com/IBM-Swift/CloudEnvironment.git", majorVersion: 4),
-<% dependencies.forEach(function(dependency) { -%>
-        <%- dependency %>
-<% }) -%>
+      .package(url: "https://github.com/IBM-Swift/Kitura.git", .upToNextMinor(from: "1.7.9")),
+      .package(url: "https://github.com/IBM-Swift/HeliumLogger.git", .upToNextMinor(from: "1.7.1")),
+      .package(url: "https://github.com/IBM-Swift/CloudEnvironment.git", .upToNextMinor(from: "4.0.5")),
+      .package(url: "https://github.com/IBM-Swift/Configuration.git", .upToNextMinor(from: "1.0.0")),
+<%  dependencies.forEach(function(dependency) { -%>
+      <%- dependency %>
+<%  }) -%>
     ],
-    exclude: ["src"]
+    targets: [
+      .target(name: "<%- executableModule %>", dependencies: [ .target(name: "<%- applicationModule %>"), "Kitura" , "HeliumLogger"]),
+      .target(name: "<%- applicationModule %>", dependencies: [ "Kitura", "Configuration", "CloudEnvironment",<% modules.forEach(function(module) { -%><%-module + "," %><%  })-%>
+<%  if (appType === 'crud') { -%>
+.target(name: "<%- generatedModule %>"),
+<%  } _%>
+<%  if (sdkTargets.length > 0) { -%>
+<%    Object.keys(sdkTargets).forEach(function(target) { -%>
+.target(name: "<%- sdkTargets[target] %>"),
+<%    }); -%>
+      ]),
```

Comment:
Is this correct, it seems like you have the `sdkTargets` in the dependencies twice? Could be a merge error, or perhaps I'm just not familiar enough with the new format?

Reply:
I first add the `sdkTargets` as `dependencies` of the `applicationModule` then I declare each `sdkTarget` with their dependency and path

Reply:
Perfect, thanks for the explanation.